misc/acrn-config/schema/VMtypes.xsd

```diff
@@ -38,33 +40,45 @@
   <xs:sequence>
 <xs:element name="guest_flag" type="GuestFlagsOptionsType" maxOccurs="unbounded"/>
   </xs:sequence>
+  <xs:assert test="count(guest_flag) eq count(distinct-values(guest_flag))"/>
 </xs:complexType>
 
 <xs:complexType name="CpuAffinityConfiguration">
   <xs:sequence>
-<xs:element name="pcpu_id" type="xs:integer" maxOccurs="unbounded"/>
+<xs:element name="pcpu_id" type="xs:integer" default="2" maxOccurs="unbounded"/>
   </xs:sequence>
 </xs:complexType>
 
 <xs:complexType name="ClosConfiguration">
   <xs:sequence>
-<xs:element name="vcpu_clos" type="xs:integer" maxOccurs="unbounded"/>
+<xs:element name="vcpu_clos" type="xs:integer" default="0" maxOccurs="unbounded"/>
   </xs:sequence>
 </xs:complexType>
 
 <xs:complexType name="EpcSection">
+  <xs:annotation acrn:configurable="n">
```

Comment:
Does a non-configurable complex type mean all its descendants are non-configurable?

Reply:
Good question, that's whether the xml did. I assume it should mean all this section is not configurable, move them to the child node attribute.
@szhen11 would this acrn:configurable="n" be in parent "EpcSection" or in child "base" and "size" change the UI behaviour?

Reply:
If acrn:configurable="n" in EpcSection, UI will not display EpcSection element and all of its child elements; If acrn:configurable="n" in "base",  UI will not display the base element; If all child elements of EpcSection are configurable="n", we should set acrn:configurable="n" for EpcSection.